gui/default/syncthing/core/syncthingController.js

```diff
@@ -581,6 +597,27 @@ angular.module('syncthing.core')
             $http.get(urlbase + '/system/config').success(function (data) {
                 updateLocalConfig(data);
                 console.log("refreshConfig", data);
+
+                //update web addres
+                for (var index in $scope.config.devices) {
+                    var device = $scope.config.devices[index];
+                    var subId = device.deviceID.substr(0, 6)
```

Comment:
Using a shorter device ID as map key doesn't seem advantageous, we generally use the full device ID.